.travis.yml

```diff
@@ -1,23 +1,16 @@
 language: julia
 os:
   - linux
+dist: xenial
+if: branch = master OR tag IS present OR type = pull_request
 julia:
   - 1.0
   - 1.3
-if: branch = master OR tag IS present OR type = pull_request
 notifications:
   email: false
 matrix:
   allow_failures:
   - julia: nightly
+codecov: true
 after_success:
-  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())';
-jobs:
-  include:
-    - stage: "Documentation"
-      julia: 1.0
-      os: linux
-      script:
-        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=".")); Pkg.instantiate()'
-        - julia --project=docs/ docs/make.jl
-      after_success: skip
+  - docs/build.sh
```

Comment:
FWIW I think we only do this for private packages because there's currently no automated documentation deployment, so viewing rendered documentation has to be done locally. IMO we can keep it as-is for public packages.

Reply:
Reverted here: https://github.com/beacon-biosignals/EDF.jl/pull/26/commits/d51965b6ea43729cbb69d00e32ab9bc04dd8ded7